spec/index.html

```diff
@@ -860,6 +867,69 @@ <h4>The <code>&lt;<dfn id="the-layer-element">layer</dfn>&gt;</code> element</h4
         <p>The location and zoom level of the map can be changed via the <a href="#dom-htmllayerelement-zoomto"><code>layer</code>.<code>zoomTo()</code></a> API method.</p>
+        <p>A layer's features may be converted to a GeoJSON-encoded feature collection [[rfc7946]]
+          with the <a href="#dom-htmllayerelement-mapml2geojson">mapml2geojson 
+          method</a>. The method accepts parameters via the 
+          <a href="#dom-htmllayerelement-mapml2geojson">dictionary-valued <code>options</code>
+          parameter</a>. Options include a <a href="https://webidl.spec.whatwg.org/#idl-callback-functions">callback</a> 
+          function, passed as the value of the 
+          <a href="#dom-geojsongenerationoptions-propertyfunction"><code>propertyFunction</code></a> 
+          option, which accepts a single <a href="#the-properties-element"><code>properties</code></a> 
+          element argument. The 
+          <a href="#dom-geojsongenerationoptions-propertyfunction"><code>propertyFunction</code></a>
+          will be invoked once for each feature in the layer, and the value of 
+          the argument passed to it will be the 
+          <a href="#the-properties-element"><code>properties</code></a> element 
+          child of the currently-processing 
+          <a href="#the-feature-element"><code>feature</code></a> element. If 
+          no <a href="#dom-geojsongenerationoptions-propertyfunction"><code>propertyFunction</code></a> 
+          option is supplied, a default mapping of properties content to the 
+          <a href="https://www.rfc-editor.org/rfc/rfc7946#section-3.2">GeoJSON "properties" member</a>
+          will be attempted, which may produce imperfect output, depending on the
+          structure of the <a href="#the-properties-element"><code>properties</code></a> 
+          element that is passed. The default 
+          <a href="#dom-geojsongenerationoptions-propertyfunction"><code>propertyFunction</code></a> 
+          attempts to reverse an HTML table that would be generated by the 
+          <a href="#dom-htmlmapmlviewerelement-geojson2mapml"><code>geojson2mapml</code></a> 
+          function.  If the input <a href="#the-properties-element"><code>properties</code></a>
+          element does not conform to that structure, incorrect output MAY result.
+        </p>
+        <p>
+          The <a href="#dom-geojsongenerationoptions-transform"><code>transform</code></a> option, 
+          allows the author to trigger the reverse projection transformation of 
+          the coordinates of feature geometries when converting to the GeoJSON 
+          [[rfc7946]] format. By default 
+          (<a href="#dom-geojsongenerationoptions-transform"><code>transform</code></a> === <code>false</code>),
+          no reverse projection transformation of coordinates will be performed. 
```

Comment:
With the current implementation, the transform option is set to 'true' by default, so it tries to do the reverse projection transformation of the coordinates, unless explicitly told not to through the transform=false option.

Should the implementation be changed (minor fix)?

Reply:
I changed the description.  That should teach me to ad lib late in the afternoon.  Thanks for catching this!